en/docs/learn/x509certificate-authenticator.md

```diff
@@ -0,0 +1,703 @@
+                     <Parameter name="priority">1</Parameter>
+                     <Parameter name="fullChainValidation">true</Parameter>
+                     <Parameter name="retryCount">2</Parameter>
+        </Validator>
+        <Validator name="org.wso2.carbon.identity.x509Certificate.validation.validator.OCSPValidator" displayName="OCSPValidator" enable="false">
+                     <Parameter name="priority">2</Parameter>
+                     <Parameter name="fullChainValidation">true</Parameter>
+                     <Parameter name="retryCount">1</Parameter>
+        </Validator>
+    </Validators>
+    </CertificateValidation>
+    ```
+
+**Disabling certificate validation in an already-started WSO2 IS pack**
+
+Follow the steps below to disable certificate validation if WSO2 Identity Server was started before.
+
+1.  Access the WSO2 Identity Server Management Console.
+2.  Click **Main &gt; Registry &gt; Browse**.  
+    ![](../assets/img/50501577/112378780.png)
+3.  Disable CRL certificate validation.
+    1.  Locate the CRL parameter by entering
+        `            _system/governance/repository/security/certificate/validator/crlvalidator           `
+        in the **Location** search box.  
+        ![](../assets/img/50501577/112378782.png)
+    2.  Expand **Properties**.  
+        ![](../assets/img/50501577/112378786.png)
+    3.  Click **Edit** pertaining to the **Enable** property.  
+        ![](../assets/img/50501577/112378794.png)  
+    4.  Change the value to `            false           ` and click
+        **Save**.  
+        ![](../assets/img/50501577/112378795.png)
+4.  Similarly, disable OCSP certificate validation in the
+    `          _system/governance/repository/security/certificate/validator/ocspvalidator         `
+    registry parameter.
+
+For more information on CRL and OCSP certificate validation, see
+[Configuring Certificate Revocation
+Validation](https://docs.wso2.com/display/ISCONNECTORS/Configuring+Certificate+Revocation+Validation)
+.
+
+## Configuring the Authentication Endpoint
+
+1.  Open the `          deployment.toml         ` file in
+    the `          <IS_HOME>/repository/conf/         `
+    directory.
+2.  Add the following configuration to the file.
+
+    1.  `            authentication_endpoint           ` : This is the
+        URL with the port that is secured with the certificate, e.g.,
+        `                         https://localhost:8443/x509-certificate-servlet                       `
+       . Update this based on your host name.
+    2.  `            username           ` : This attribute value will be
+        taken as the authenticated user subject identifier. Update this
+        with any of the certificate attributes, e.g., CN and Email.
+
+    ``` toml
+    [authentication.authenticator.x509_certificate.parameters]
+    name ="x509CertificateAuthenticator"
+    enable=true
```

Comment:
shall we add descriptions for these two parameters as well?

Reply:
Added